### DISCUSSION TITLE
How to fix VRAM volume...

### DISCUSSION THREAD

**Discussion by divinets** [lllyasviel/Fooocus#3680] [Q&A] [answered]
CPU: AMD Ryzen 5800H, GPU: AMD Radeon RX 6600 M (8 GB)

I switched off internal GPU and I have only one GPU on the system. I run with a command: python entry_with_update.py --directml --preset realistic. And every time VRAM is 1024 MB. Why? Any recommendation?

```
Already up-to-date
Update succeeded.
[System ARGV] ['entry_with_update.py', '--directml', '--preset', 'realistic']
Python 3.12.7 (tags/v3.12.7:0b05ead, Oct  1 2024, 03:06:41) [MSC v.1941 64 bit (AMD64)]
Fooocus version: 2.5.5
Loaded preset: C:\Users\M\Fooocus\presets\realistic.json
[Cleanup] Attempting to delete content of temp dir C:\Users\M\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Using directml with device:
Total VRAM 1024 MB, total RAM 15755 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: privateuseone
VAE dtype: torch.float32
```

**Comment by divinets** [comment]
![Clip2net_241015175259](https://github.com/user-attachments/assets/296878f7-ee07-4b71-9aea-5f2ae3cf1ff8)

**Comment by divinets** [comment]
I'm trying to use with both GPU - same result.

**Comment by divinets** [comment]
![Clip2net_241015175259](https://github.com/user-attachments/assets/d4c1176a-46c9-4000-8032-91664736bbc8)

**Comment by Linapse** [comment]
I have the same problem. I have no idea what's wrong

**Comment by NoamTheStudent** [comment]
I also have the same problem

**Comment by griede26-ui** [comment, marked as answer]
if you switched from an internal Gpu to a new dedicated GPU, you need to update several Bios Settings. inorder for your system to properly recignize the new card.

also check to be sure you have the latest Drivers for your video card.
